### ISSUE TITLE
Consistency mode not passed in request

### ISSUE THREAD

**Issue by antoine-galataud** [OrbitzWorldwide/consul-client#85]
Hi,

Here is a sample of consul service discovery query using stale consistency mode:

```java
CatalogOptions cops = ImmutableCatalogOptions.builder()
                .datacenter(ConsulUtils.getDataCenter()).tag(ConsulUtils.getServiceTag()).build();
QueryOptions queryOpts = ImmutableQueryOptions.builder()
            .consistencyMode(ConsistencyMode.STALE).build();

ConsulResponse<List<ServiceHealth>> resp = myClient.healthClient()
                .getHealthyServiceInstances(lab, cops, queryOpts);
```

In [QueryOptions](https://github.com/OrbitzWorldwide/consul-client/blob/master/src/main/java/com/orbitz/consul/option/QueryOptions.java), stale query parameter is added like this:
```java
added = added.queryParam("stale");
```

This doesn't seem to work: as stated in jersey [doc](https://jersey.java.net/apidocs/2.0/jersey/org/glassfish/jersey/client/JerseyWebTarget.html#queryParam(java.lang.String, java.lang.Object...)), passing null values removes the parameter from the query.

Am I missing something?

**Comment by rickfast** [comment]
Looks like a bug. I'll fix. Thanks!

**Comment by rickfast** [comment]
fixed by e22bfd084d000627a54d9bbfc81608d5e3ac6dfa
